chains/iliad/metadata.yaml

```diff
@@ -0,0 +1,22 @@
+# yaml-language-server: $schema=../schema.json
+blockExplorers:
+  - apiUrl: https://testnet.storyscan.xyz/api
+    family: blockscout
+    name: StoryScan
+    url: https://testnet.storyscan.xyz
+blocks:
+  confirmations: 1
+  estimateBlockTime: 4
+  reorgPeriod: 0
+chainId: 1513
+displayName: Iliad Testnet
```

Comment:
could you also add deployer details to iliad + euphoria metadata please 🙏 

Example here:
https://github.com/hyperlane-xyz/hyperlane-registry/blob/2247762c1b8f96e5a37cbf69481293dc0d84f035/chains/ancient8/metadata.yaml#L12-L14

Reply:
updated